source/_patterns/01-molecules/15-loan-sections/18-loan-story.mustache

```diff
@@ -67,7 +67,7 @@
 					<img class="reviewer-image" src="{{reviewerImageLink}}" alt="Translator profile picture"/>
 				{{/reviewerImageLink}}
 				<p>
-					Translated from {{language}} by {{{reviewerLink}}} {{reviewerName}}
+					Translated from {{language}} {{#reviewerLink}} by {{{reviewerLink}}}{{/reviewerLink}} {{reviewerName}}
```

Comment:
@coderintherye - I think this is not quite right. "by" seems to go with {{reviewerName}} - which is outside the condition. I think {{reviewerName}} should go inside the condition too. {{{reviewerLink}}} is just "Kiva translator" with a link to the translation program page.